.circleci/config.yml

```diff
@@ -33,6 +33,15 @@ node-core-base: &node-core-base
       run:
         name: Unit tests
         command: yarn test:core:ci
+    - run:
+        name: Upload JUnit XML Report
+        command: |
+          if [ -n "$CIRCLE_PR_NUMBER" ]; then
```

Comment:
inspired by https://circleci.com/blog/managing-secrets-when-you-have-pull-requests-from-outside-contributors/#:~:text=To%20enable%20CircleCI%20for%20forked,builds%20from%20forked%20pull%20requests